### PR TITLE
Enable frontend test in openshift-pipelines console-plugin

### PR DESCRIPTION
**Pull request by vikram-raj**
Enable unit tests in `openshift-pipelines/console-plugin` repo and `jira/valid-bug` and `jira/valid-reference` as required labels to merge the PRs.

### CODE REVIEW

File: core-services/prow/02_config/openshift-pipelines/console-plugin/_prowconfig.yaml [hunk 1]
```diff
@@ -3,6 +3,8 @@ tide:
   - labels:
     - approved
     - lgtm
+    - jira/valid-bug
+    - jira/valid-reference
```
Comment (jerolimov) [inline core-services/prow/02_config/openshift-pipelines/console-plugin/_prowconfig.yaml:7]: Hey, as said on slack, this adds the requirement that every PR contains both labels which isn't the case for features.

If you want that the `tide` job passes with one of both labels you need two queries as we use in the [openshift/console/_prowconfig.yaml](https://github.com/openshift/release/blob/master/core-services/prow/02_config/openshift/console/_prowconfig.yaml)

It's up to you if you accept valid-bug + valid-reference for bugs and then only valid-reference for features in two queries. Or you remove the valid-bug for now until features and bugs has other requirments.

```suggestion
    - jira/valid-reference
```

**Review by jerolimov — CHANGES_REQUESTED** [review]
The other changes to add unit tests are fine. But the prowconfig change will only accept bugs:

File: core-services/prow/02_config/openshift-pipelines/console-plugin/_prowconfig.yaml (hunk 1 above)
Comment (vikram-raj) [reply core-services/prow/02_config/openshift-pipelines/console-plugin/_prowconfig.yaml]: updated it.

**Comment by vikram-raj** [comment]
/pj-rehearse pull-ci-openshift-pipelines-console-plugin-main-frontend

**Comment by jerolimov** [comment]
/pj-rehearse pull-ci-openshift-pipelines-console-plugin-release-4.16-frontend

**Comment by jerolimov** [comment]
/pj-rehearse pull-ci-openshift-pipelines-console-plugin-main-e2e
/pj-rehearse pull-ci-openshift-pipelines-console-plugin-main-frontend

**Comment by vikram-raj** [comment]
/pj-rehearse pull-ci-openshift-pipelines-console-plugin-main-frontend

**Comment by vikram-raj** [comment]
/pj-rehearse pull-ci-openshift-pipelines-console-plugin-main-e2e

**Comment by vikram-raj** [comment]
/pj-rehearse pull-ci-openshift-pipelines-console-plugin-release-4.15-e2e

**Comment by vikram-raj** [comment]
/pj-rehearse pull-ci-openshift-pipelines-console-plugin-release-4.17-frontend

**Pull request merged**